images/win/windows2022.json

```diff
@@ -143,7 +151,7 @@
         },
         {
             "type": "windows-restart",
-            "restart_timeout": "10m"
+            "restart_timeout": "30m"
```

Comment:
Just curious, why this change?

Reply:
During the image generation several attempts were unsuccessful due to short reboot timeout: `vhd: Timeout waiting for machine to restart.`

Reply:
Ok. Thanks! Interesting that test-signing caused that. Oh well!